lib/hex_web/package.ex

```diff
@@ -202,6 +203,20 @@ defmodule HexWeb.Package do
   defp like_escape(string, escape) do
     String.replace(string, escape, "\\\\\\1")
   end
+
+  defp sort(query, nil) do
+    sort(query, :downloads)
+  end
```

Comment:
I see from the code below that we actually need this clause. Change the code below so that `nil => :name`.